src/vnm_mad/remotes/lib/security_groups_iptables.rb

```diff
@@ -334,6 +334,14 @@ def self.vars(vm, nic, sg_id = nil)
             vars[:set_sg_out] = "#{vars[:chain]}-#{sg_id}-o"
         end
 
+        vars[:nic_aliases] = []
+
+        unless nic[:alias_ids].nil?
```

Comment:
prefer if !nic[:alias_ids].nil? for single statements

Reply:
I will change it